Cargo.toml

```diff
@@ -18,15 +24,21 @@ bytes = "1"
 futures = "0.3"
 http = "0.2"
 pin-project = "1"
-reqwest = { version = "0.11.22", default-features = false, features = ["json", "rustls-tls", "stream"] }
+reqwest = { version = "0.11.22", default-features = false, features = [
+  "json",
+  "rustls-tls",
+  "stream",
+] }
 rodio = { version = "0.17.0", optional = true }
 serde = { version = "1.0.25", features = ["derive"] }
 serde_json = "1"
 serde_urlencoded = "0.7"
 thiserror = "1"
 tokio = { version = "1.38.0", features = ["full"] }
 tokio-stream = "0.1.15"
-tokio-tungstenite = { version = "0.20.1", features = ["rustls-tls-webpki-roots"], optional = true }
+tokio-tungstenite = { version = "0.20.1", features = [
+  "rustls-tls-webpki-roots",
+], optional = true }
```

Comment:
```suggestion
tokio-tungstenite = { 
  version = "0.20.1",
  features = ["rustls-tls-webpki-roots"], 
  optional = true
}
```

Reply:
This looks a bit nicer to me than splitting the features list onto multiple lines but inlining the arguments